cluster/gce/configure-vm.sh

```diff
@@ -245,14 +245,34 @@ admission_control: '$(echo "$ADMISSION_CONTROL" | sed -e "s/'/''/g")'
 EOF
```

Comment:
Makes sense, but that sucks that the subshell doesn't inherit the `nounset` since it would then cause the `noerr` clause to be hit here and we could discover when one of these vars isn't being set as desired. Guess we'll just have to use our eyes instead.